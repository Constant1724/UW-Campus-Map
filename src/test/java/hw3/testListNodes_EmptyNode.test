# START_TEST
# NAME EmptyNode
# Create a graph
CreateGraph graph1

# Add No nodes

# Print the nodes in the graph
ListNodes graph1
# END_TEST