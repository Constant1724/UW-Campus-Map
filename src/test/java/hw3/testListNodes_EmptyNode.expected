# START_TEST
# NAME EmptyNode
# Create a graph
created graph graph1

# Add No nodes

# Print the nodes in the graph
graph1 contains:
# END_TEST